<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.3">
    <title>Links for errordetail</title>
  </head>
  <body>
    <h1>Links for errordetail</h1>
<a href="https://files.pythonhosted.org/packages/59/18/1e114da0ab54d7d66ab4ce0fce2d02a14f30bb935d2fb4a23a05f8d70186/errordetail-1.1.1.tar.gz#sha256=a4bf324ddce06e10e84de0032962b6414423941c4174c182a8df87097f0c7b3a" data-requires-python="&gt;=3.*.*" >errordetail-1.1.1.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/d3/93/5472d646d3bb9fec609c19af80d7d08e498322677503d76ce02a2ab4c82e/errordetail-1.2.0-py3-none-any.whl#sha256=e3f98f4385902665be3ac2303dc5ab4c28d81258a903e1c8dd5667465db462d7" data-requires-python="&gt;=3.*.*" data-dist-info-metadata="sha256=0f7728cd252a459c2074f9bc28cf53f7c9b41e45ebdc0da244d33806c8caf040" data-core-metadata="sha256=0f7728cd252a459c2074f9bc28cf53f7c9b41e45ebdc0da244d33806c8caf040">errordetail-1.2.0-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/a8/6e/7664b8717e84babce458824f4e5bdaacbd77b4a3a8d74f0b183902c31881/errordetail-1.2.0.tar.gz#sha256=6d4cd11204a71c73bd734a0d176858abf102184e940be643f225752d80514055" data-requires-python="&gt;=3.*.*" >errordetail-1.2.0.tar.gz</a><br />
</body>
</html>
<!--SERIAL 14807875-->